### PR TITLE
Adding command for Filtering and print trades

### LINKED ISSUE #5121: Backtesting: Filter for Pairs with TotalProfit/Win%
## Describe your environment
(if applicable)

  * Operating system: Ubuntu SMP
  * Python Version: Python 3.8.5
  * CCXT version: not found
  * Freqtrade Version: freqtrade docker-e2bbc0aa

## Describe the enhancement

Extension for backtesting with additional pairlist filter parameters `--list-total-profit-percent="<0"` or `--list-win-percent=">95"` . The Filter could correspond to the Reportcolumns.

 This filters then outputs a pair list which could be easy transferred into the config as black or white list, depends on Filter.

Outputline could be look like this in den end of the Backtesting Report:

```
list-total-profit-percent: Choosen Pairlist Filter *Total Profit percent* corresponds "<0":
"BNB/BUSD",
"JUV/BUSD",
"OM/BUSD",
"ADA/BUSD",
"PERP/BUSD",
"AERGO/BUSD",
"AUCTION/BUSD"
```

### PR DESCRIPTION
**Pull request by cryptomeisternox**
* Read latest Backtest file and print trades

I just want to know if it is going in the right direction.

## Summary
closes #5121

## Quick changelog

- Adding command for Filtering
- Print trade as text

### CODE REVIEW

File: freqtrade/commands/cli_options.py [hunk 1]
```diff
@@ -202,6 +202,11 @@ def __init__(self, *args, **kwargs):
         help='Specify additional lookup path for Hyperopt and Hyperopt Loss functions.',
         metavar='PATH',
     ),
+    "backtest_path": Arg(
```
Comment (xmatthias) [inline freqtrade/commands/cli_options.py]: I'm ... not quite sure what you have in mind with this (doesn't seem to be used beyond this point) - but maybe `exportfilename` covers this ... ?

**Review by xmatthias — COMMENTED** [review]
For now, i don't see it filtering (so it's no different than loading + reprinting the results) - but i guess that's "still to come".

As such, i guess you're on the right track - at least for now i can't see anything terribly off (assuming it's working the way you expect it to, obviously ...) 

I've marked it as "WIP" for now - let me know when ready eventually (or when it's worth to have another intermediate look)

File: freqtrade/commands/cli_options.py (hunk 1 above)
Comment (cryptomeisternox) [reply freqtrade/commands/cli_options.py]: I had built a small text "parser" locally for the backtesting logfile and assumed I would build the filter on the logfile. That was before I realized that a yaml file was available for processing. I will think about this on my next codingrun and possible will remove this.

**Review by xmatthias — APPROVED** [review]
i took the liberty to finish this PR as it had a long time of inactivity.

Nice enhancement :+1:

**Pull request merged**